src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -205,3 +235,73 @@ const getPublicness = (currentTabName: typeof TABS[number]) => {
       return undefined
   }
 }
+
+export const useDeleteVideo = (memberId: string | null) => {
```

Comment:
not exactly sure if we should keep it here. Maybe we should keep the consistency and move it to hooks folder